src/NServiceBus.AcceptanceTesting/Support/RunSettings.cs

```diff
@@ -7,15 +7,19 @@ namespace NServiceBus.AcceptanceTesting.Support
 
     public class RunSettings : IEnumerable<KeyValuePair<string, object>>
     {
-        public RunSettings()
+        public TimeSpan? TestExecutionTimeout
```

Comment:
Couldn't find any downstreams that use this so is it fair to say that this breaking change is not going to cause pain?